blockprotocol/blockprotocol
# Display Block Variants in Block Protocol

> Previous PR: https://github.com/hashintel/hash/pull/279

## What this Does

This PR adds a UI for displaying block variants and switching between them in the demo page for blocks on the blockprotocol website. This allows users to switch between variants of blocks, and check their differences, along with examples provided where available. At the time of this PR, only the embed and header blocks have variants associated with them.

Changes described in GH Comments below.

## How to test this

- Visit either the embed or header block
- Switch between variants and check if the data source and block have updated.

![chrome_sNBuvIaqCV](https://user-images.githubusercontent.com/22196279/152978283-575a05a8-600f-4a57-aa88-fa0d2f4476a1.png)
![chrome_zW8CMaqgLN](https://user-images.githubusercontent.com/22196279/152978289-9614d5a3-81c1-4421-92ba-eb5b20c31b88.gif)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201540186206729
